src/gsi/read_prepbufr.f90

```diff
@@ -149,8 +149,10 @@ subroutine read_prepbufr(nread,ndata,nodata,infile,obstype,lunout,twindin,sis,&
 !   2020-05-04  wu      - no rotate_wind for fv3_regional
 !   2020-09-05  CAPS(C. Tong) - add flag for new vadwind obs to assimilate around the analysis time only
 !   2023-03-23  draper  - add code for processing T2m and q2m for global system
-!   2023-07-30  Zhao    - added code to extract obs of significant wave height (howvob) from bufr record 
+!   2023-07-30  zhao    - added code to extract obs of significant wave height (howvob) from bufr record 
```

Comment:
General comment:  Since our code is version controlled we do not need to add entries to the program history log.  We can use git commands to see what, when, and who changed code.

Reply:
@RussTreadon-NOAA
I was wondering why there are no more these entries for the recent GSI updates. Now I see.
Thank you!
-Gang